docs/storefront/graphql/examples/products.mdx

```diff
@@ -598,6 +598,159 @@ You can query product images at different resolutions. The following query retri
   </Tab>
 </Tabs>
 
+## Get product videos
+
+Query the product videos and all configurable media types for products. The API returns images uploaded in the control panel, image URLs added to a product using the control panel, and video URLs linked to a product. 
+
+The following example retrieves video titles and URLs associated with videos attached to a product. 
+
+<Callout type="info">
+Before performing this example, you must ensure to add videos to the product node.
```

Comment:
Do we need this? It's normal for APIs to only return data if you have added data to begin with.

Reply:
If you don't add videos, you won't receive any product video URLs. I can remove it this sentence.